src/descriptor.js

```diff
@@ -1,16 +1,10 @@
 const debug = require('debug')('restinga:descriptor');
 const {version} = require('../package.json');
 
-export default class Descriptor {
-	/**
-	 * @param {Object} params
-	 */
-	constructor({service, prefix, agent}) {
-		debug(`Creating Descriptor ${service}`);
-
-		this.service = service;
-		this.prefix = prefix;
-		this.agent = agent || `restinga-node/${version} (https://github.com/jay-ess/restinga-node)`;
-	}
-}
+module.exports = function ({service, prefix, agent}) {
+	this.service = service;
```

Comment:
Don't do `this` in a function.

Prefer to return a object.

Reply:
understood!